org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/ScanPointGeneratorFactoryTest.java

```diff
@@ -216,7 +216,7 @@ public void testJCompoundGeneratorFactoryWithRaster() {
 		JythonObjectFactory<ScanPointIterator> compoundGeneratorFactory = ScanPointGeneratorFactory.JCompoundGeneratorFactory();
 
         Object[] generators = {outerLine, innerLine};
-        Object[] excluders = CompoundSpgIterator.getExcluders(Arrays.asList(new ScanRegion<>(new RectangularROI(0,0,5,5,0), Arrays.asList("x", "y"))));
+        Object[] excluders = CompoundSpgIteratorFactory.getExcluders(Arrays.asList(new ScanRegion<>(new RectangularROI(0,0,5,5,0), Arrays.asList("x", "y"))));
         Object[] mutators = {};
 
         ScanPointIterator iterator = (ScanPointIterator)  compoundGeneratorFactory.createObject(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "ScanPointIterator". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)